### PR TITLE
refactor headers

### PR DESCRIPTION
**Pull request by shuisman**
Reviewers: @usabilla/plumbers @usabilla/feds 

### Changelog:
- split files
- refactor headers for DRY
- refactor headers for easy adding headers

### CODE REVIEW

File: src/resources.js [hunk 1]
```diff
@@ -40,6 +38,7 @@ class Resource {
     var _results = results || [];
 
     this.signatureFactory.setUrl(this.url);
+    this.signatureFactory.setHeaders({'user-agent': 'Usabilla API Client for Node.js'});
```
Comment (shuisman) [inline src/resources.js]: any suggestions?

File: src/resources.js [hunk 2]
```diff
@@ -1,6 +1,4 @@
 "use strict";
```
Comment (shuisman) [inline src/resources.js]: is this normal?

Comment (tzengerink) [inline src/resources.js]: I consider it normal, yes. See [MDN article on Strict mode](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Strict_mode)

File: src/resources.js (hunk 1 above)
Comment (tzengerink) [inline src/resources.js]: Good be nice to add the version of the client (which you could grab from the package.json).

Maybe something like `Usabilla API Node Client/1.0.0-beta.1`

**Pull request merged**